src/App.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'
+import AddTodo from './components/AddTodo'
+import Hints from './components/Hints'
+import Title from './components/Title'
+import TodoInput from './components/TodoInput'
+import TodosList from './components/TodosList'
+import Wrapper from './components/Wrapper'
+
+function App() {
+  const [showInput, setShowInput] = useState(false)
+  const [todos, setTodos] = useState([]);
+
+  const addTodo = (title) =>{
+    setTodos(v => [...v, {id: (v.at(-1)?.id ?? 0) + 1, title, done: false}])
+  };
+
+  const deleteTodo = (id) =>{
+    setTodos(v => v.filter(todo => todo.id !== id))
+  };
+
+  const toggleTodoStatus = (id) =>{
+    const todo = todos.find(td => td.id === id);
+    setTodos(v => [...v.filter(td => td.id !== id), {...todo, done: !todo.done}].sort((a, b) => a.id - b.id))
```

Comment:
I think wherever you have this v, can be good if you change it to be something more descriptive... for example prevTodo or something else instead of this v.